cpu/kinetis/periph/pwm.c

```diff
@@ -106,13 +141,40 @@ uint32_t pwm_init(pwm_t pwm, pwm_mode_t mode, uint32_t freq, uint16_t res)
                        PORT_PCR_MUX(pwm_config[pwm].chan[i].af));
         /* set the given mode */
         ftm(pwm)->CONTROLS[pwm_config[pwm].chan[i].ftm_chan].CnSC = mode;
-        /* and reset the PWM to 0% duty cycle */
-        ftm(pwm)->CONTROLS[pwm_config[pwm].chan[i].ftm_chan].CnV = 0;
```

Comment:
Why was this removed?

Reply:
Whoops. I neglected to grab the latest pwm.c from my branch before pushing this. I'll push a fixup that adds this back along with other small changes.

Incidentally, this was the beginning of my attempt to eliminate flickering during init by using a `pwm_set(); pwm_init()` paradigm. It didn't work because we can't `pwm_set()` while the hardware is declocked. I'll deal with the flicker in another PR.

Reply:
> I'll deal with the flicker in another PR.

For now I made #14721 to track this.